tests/test.rfb.js

```diff
@@ -4030,6 +4031,21 @@ describe('Remote Frame Buffer protocol client', function () {
                 expect(pointerEvent.secondCall).to.have.been.calledWith(client._sock,
                                                                         10, 10, 0);
             });
+
+            it('should handle wheel event with buttons pressed', function () {
```

Comment:
Can this be moved earlier? So we can see that your changes just kept existing behaviour.